lib/helpers/utils.js

```diff
@@ -5004,6 +5005,35 @@ export async function getGoPkgLicense(repoMetadata) {
   return undefined;
 }
 
+/**
+ * Method to get go pkg vcs url from go.dev site.
+ *
+ * @param {String} group Package group
+ * @param {String} name Package name
+ */
+async function getGoPkgVCSUrl(group, name) {
+  const fullName = getGoPkgFullName(group, name);
+  if (fullName.startsWith("github.com") || fullName.startsWith("gitlab.com")) {
+    return `https://${fullName}`;
+  }
+  const pkgUrl = getGoPkgUrl({ fullName });
+  if (metadata_cache[pkgUrl]) {
```

Comment:
Can we use a different cache here? I think this might be overwriting the other metadata cache?

Reply:
Different keys are used: in getGoPkgLicense - pkgUrl?tab=licenses, in getGoPkgVCSUrl - pkgUrl. The keys don't overlap.